front/lib/registry.ts

```diff
@@ -336,9 +275,42 @@ export const DustProdActionRegistry = createActionRegistry({
       },
     },
   },
-});
+};
+
+export type DustRegistryActionName = keyof typeof BaseDustProdActionRegistry;
 
-export type DustRegistryActionName = keyof typeof DustProdActionRegistry;
+let dustProdActionRegistry: ActionRegistry | undefined;
+
+export const getDustProdActionRegistry = () => {
+  if (!dustProdActionRegistry) {
+    const workspaceId = config.getDustAppsWorkspaceId();
+    const appSpaceId = config.getDustAppsSpaceId();
+
+    dustProdActionRegistry = Object.entries(
```

Comment:
`nit` to remove the cast of the accumulator!
```suggestion
    dustProdActionRegistry = Object.entries<ActionRegistry>(
```

Reply:
this would be something like  Object.entries<{app: .., config:..}> , but that won't help the reducer . I did not find a way to get rid of the cast